app/javascript/components/Products/Checkout/index.js

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+import './style.css';
+
+const CheckoutPage = (props) => {
+
+  return (
+    <div className="payment-page">
+      <h2>Pagamento</h2>
+      <div className="row">
+        <div className="col-75">
+          <div className="container">
+            <form>
+              <div className="row">
+                <div className="col-50">
+                  <h3>Endereço de Cobrança</h3>
+                  <label for="name"><i className="fa fa-user"></i> Nome Completo</label>
+                  <input
+                    type="text"
+                    id="name"
+                    name="firstname"
+                    placeholder="Joao da Silva"
+                  />
+                  <label for="email"><i className="fa fa-envelope"></i> Email</label>
+                  <input
+                    type="text"
+                    id="email"
+                    name="email"
+                    placeholder="john@exemplo.com"
+                  />
+                  <label for="end"
```

Comment:
`<label for="end">`

Aqui e a baixo também acontece, mas coloca a tag de fechamento na mesma linha quando não tem muitos atributos dentro da tag, tipo aqui so tem o `for`